quizapp/src/test/groovy/billennium/quizapp/QuizSpec.groovy

```diff
@@ -0,0 +1,118 @@
+package billennium.quizapp
+
+import billennium.quizapp.controller.QuizController
+import billennium.quizapp.entity.*
```

Comment:
jw

Reply:
jeżeli prosisz go o zmiane taką wytłumacz dlaczego tak.

Reply:
prosty przykład nie raz spotkałem się z tym że jeśli jest import z * idea sama wrzuca import do tej samej grupy mimo że czasami chcemy użyć czegoś z innej biblioteki. Niejednokrotenie spotkałem się z tym że był jakiś problem problem z działaniem bo był zły import biblioteki i nawet bo nie było widać bo wszystko było schowane w import *
Jak to zmienić: 
File > Settings > Project Settings > Code Style > Java > Imports > General > Class count to use import with '*'  
Moim zdaniem każdy import powinien być zadeklarowany jawnie bez opakowywania go w *